src/aws-api-mcp-server/awslabs/aws_api_mcp_server/server.py

```diff
@@ -44,10 +47,27 @@
 from typing import Annotated, Any, Optional, cast
 
 
-# Configure Loguru logging
+def _get_log_directory():
+    """Get platform-appropriate log directory"""
+    base_location = 'aws-api-mcp'
+    if os.name == 'nt':  # Windows
+        return Path(tempfile.gettempdir()) / base_location
+    elif os.uname().sysname == 'Darwin':  # macOS
+        return Path(tempfile.gettempdir()) / base_location
+    # Linux
+    base_dir = os.environ.get('XDG_RUNTIME_DIR') or os.environ.get('TMPDIR') or '/tmp'
```

Comment:
```suggestion
    base_dir = os.environ.get('XDG_RUNTIME_DIR') or os.environ.get('TMPDIR') or tempfile.gettempdir()
```
I think this will get rid of the warning